src/main/scala/zio/package.scala

```diff
@@ -28,21 +28,20 @@ import zio.zmx.diagnostics.fibers.FiberDumpProvider
 import zio.zmx.diagnostics.parser.ZMXParser
 import zio.zmx.metrics._
 
-import scala.collection._
-import scala.collection.immutable.SortedSet
+import scala.collection.immutable.HashSet
 import zio.zmx.diagnostics.graph.{ Edge, Graph, Node }
 
 package object zmx extends MetricsDataModel with MetricsConfigDataModel {
 
-  val ZMXSupervisor: Supervisor[SortedSet[Fiber.Runtime[Any, Any]]] =
-    new Supervisor[SortedSet[Fiber.Runtime[Any, Any]]] {
+  val ZMXSupervisor: Supervisor[HashSet[Fiber.Runtime[Any, Any]]] =
```

Comment:
I would just use `Set` here. There is no reason it has to be a `HashSet` in particular and this will give us more flexibility.